virttest/libvirtd_decorator.py

```diff
@@ -0,0 +1,70 @@
+"""
+A decorator utility functions to apply libvirtd functions.
+
+:copyright: 2008-2009 Red Hat Inc.
+"""
+
+import logging
+import os
+from virttest import data_dir
+from virttest import libvirt_version
+
+
+def get_libvirtd_split_enable_bit():
+    base_cfg_path = os.path.join(data_dir.get_shared_dir(), 'cfg', 'base.cfg')
+    if os.path.isfile(base_cfg_path):
+        with open(base_cfg_path, 'r') as base_file:
+            for line in base_file:
+                if 'enable_split_libvirtd_feature' in line and 'yes' in line:
+                    return True
+    else:
+        logging.info("CAN NOT find base.cfg file")
+    return False
+
+
+LIBVIRD_SPLIT_ENABLE_BIT = get_libvirtd_split_enable_bit()
+IS_LIBVIRTD_SPLIT_VERSION = libvirt_version.version_compare(5, 6, 0)
```

Comment:
Calling `libvirt_version.version_compare()` at the module level will *require* people to have `virsh` installed simply to load this module.  This seems to be the reason for the failures done at `avocado vt-bootstrap` which I guess is indirectly loading this module.

Reply:
yes, libvirt_version.version_compare() need install virsh firstly.
And it sound like containerized fedora and centos images in cirrcus CI doesn't install virsh

Reply:
Could dependency on `virsh` be avoided?
```bash
# /usr/sbin/libvirtd --version
/usr/sbin/libvirtd (libvirt) 6.0.0
```